### PR TITLE
feat(sbb-select): introduce size s

### LINKED ISSUE #2763: Select: Size S
https://www.figma.com/file/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?type=design&node-id=30169%3A346979&mode=design&t=aBVGHDIkLF2bjfjV-1

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/sbb-design-systems/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #2763 

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../docs/REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- minor changes to sbb-form-field with size s to support sbb-select

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [x] Chrome Desktop
- [ ] Edge Desktop
- [x] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/elements/select/select.stories.ts [hunk 1]
```diff
@@ -49,6 +49,16 @@ const borderless: InputType = {
   },
 };
 
+const size: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['m', 's'],
+  table: {
+    category: 'Form-field',
```
Comment (TomMenga) [inline src/elements/select/select.stories.ts:58]:
```suggestion
    category: 'Form field',
```

File: src/elements/select/readme.md [hunk 2]
```diff
@@ -70,6 +70,18 @@ the selected values will be displayed in selection order, separated by a comma.
 </sbb-form-field>
 ```
 
+## Style
+
+The component has no `size` property but, when slotted in a `sbb-form-field`,
+the height of the panel's trigger adapts based on the parent `size` property.
```
Comment (TomMenga) [inline src/elements/select/readme.md:76]:
```suggestion
it adapts to the parent `size` property.
```
Too technically deep imho

Comment (TomMenga) [reply src/elements/select/readme.md:76]: Maybe "The `sbb-select` adapts its size to the `sbb-form-field`" would be enough

**Review by TomMenga — APPROVED** [review]
LGTM 👍 
Just some minor things

File: src/elements/form-field/form-field/form-field.scss [hunk 3]
```diff
@@ -388,7 +388,14 @@
 .sbb-form-field__input {
   display: flex;
 
-  :host([size='s']:is([data-input-type='input'], [data-input-type='select'])) & {
+  :host(
```
Comment (jeripeierSBB) [inline src/elements/form-field/form-field/form-field.scss:391]: Shouldn't the sbb-options not also receive styles for size s?

File: src/elements/select/select.visual.spec.ts [hunk 4]
```diff
@@ -194,6 +196,20 @@ describe('sbb-select', () => {
...
 
+        it(
+          `negative=${negative} multiple=${multiple} size=s`,
+          visualDiffDefault.with(async (setup) => {
+            await setup.withFixture(template({ ...defaultArgs, negative, multiple, size: 's' }), {
+              minHeight: '600px',
+              backgroundColor: negative ? 'var(--sbb-color-black)' : undefined,
+            });
+            setup.withPostSetupAction(() => {
+              const select = setup.snapshotElement.querySelector('sbb-select')!;
```
Comment (jeripeierSBB) [inline src/elements/select/select.visual.spec.ts:207]: I think there should be a selected option to show placement of the label in the form field size s

File: src/elements/form-field/form-field/form-field.scss (hunk 3 above)
Comment (DavideMininni-Fincons) [reply src/elements/form-field/form-field/form-field.scss:391]: Comparing a screenshot with the design, I see no differences in sbb-options dimensions/style
image

File: src/elements/select/select.visual.spec.ts (hunk 4 above)
Comment (DavideMininni-Fincons) [reply src/elements/select/select.visual.spec.ts]: Just pushed a fix for it

File: src/elements/form-field/form-field/form-field.scss (hunk 3 above)
Comment (jeripeierSBB) [reply src/elements/form-field/form-field/form-field.scss:391]: sorry for the confusion then, and thanks for checking it!!

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**